### PR TITLE
Fix axis position compared to tasks position

### PR DESCRIPTION
**Pull request by adiletelf**

### CODE REVIEW

File: src/gantt.ts [hunk 1]
```diff
@@ -328,7 +328,9 @@ export class Gantt implements IVisual {
     private static TaskOpacity: number = 1;
     public static RectRound: number = 7;
 
+
     private static TimeScale: timeScale<any, any>;
+    private static xAxisProperties: IAxisProperties;
```
Comment (MulyukovAidar) [inline src/gantt.ts:333]: static?

File: src/gantt.ts [hunk 2]
```diff
@@ -2010,7 +2009,7 @@ export class Gantt implements IVisual {
         const categoriesAreaBackgroundColor: string = this.colorHelper.getThemeColor();
         const isHighContrast: boolean = this.colorHelper.isHighContrast;
 
-        this.updateCollapseAllGroup(categoriesAreaBackgroundColor);
+        this.updateCollapseAllGroup(categoriesAreaBackgroundColor, taskLabelsShow);
```
Comment (MulyukovAidar) [inline src/gantt.ts:2012]: see it in action

File: src/gantt.ts [hunk 3]
```diff
@@ -3104,13 +3108,13 @@ export class Gantt implements IVisual {
...
         this.axisGroup
-            .attr("transform", SVGManipulations.translate(settings.taskLabelsCardSettings.width.value + margin.left + Gantt.SubtasksLeftMargin, translateYValue));
+            .attr("transform", SVGManipulations.translate(translateXValue, translateYValue));
 
         translateXValue = (this.ganttDiv.node() as SVGSVGElement).scrollLeft;
         this.lineGroup
             .attr("transform", SVGManipulations.translate(translateXValue, 0));
         this.collapseAllGroup
-            .attr("transform", SVGManipulations.translate(0, margin.top / 4));
+            .attr("transform", SVGManipulations.translate(0, margin.top / 4 + 6));
```
[the remark this answers is not in the document]
Comment (adiletelf) [reply src/gantt.ts:3117]: removed

File: src/gantt.ts (hunk 1 above)
Comment (adiletelf) [reply src/gantt.ts:333]: removed `static`

File: pbiviz.json [hunk 4]
```diff
@@ -1,10 +1,10 @@
 {
   "visual": {
     "name": "Gantt",
-    "displayName": "Gantt 3.0.7.0",
+    "displayName": "Gantt 3.0.9.0",
     "guid": "Gantt1448688115699",
     "visualClassName": "Gantt",
-    "version": "3.0.7.0",
+    "version": "3.0.8.0",
```
[the remark this answers is not in the document]
Comment (adiletelf) [reply pbiviz.json:7]: fixed

**Pull request merged**